model/adjuster/adjuster.go

```diff
@@ -5,57 +5,38 @@
 package adjuster
 
 import (
-	"errors"
-
 	"github.com/jaegertracing/jaeger/model"
 )
 
 // Adjuster applies certain modifications to a Trace object.
 // It returns adjusted Trace, which can be the same Trace updated in place.
 // If it detects a problem with the trace that prevents it from applying
-// adjustments, it must still return the original trace, and the error.
+// adjustments, it must still return the original trace.
 type Adjuster interface {
-	Adjust(trace *model.Trace) (*model.Trace, error)
+	Adjust(trace *model.Trace) *model.Trace
```

Comment:
Any reason to return the trace? I don't think any adjusters actually create a new one, they always return the original trace that may be changed in place. So I would make it consistent with the v2 Adapter API to simplify things.

Reply:
@yurishkuro was thinking the same thing as well - done